### PR TITLE
FE-10: Login/Signup Pages

### PR DESCRIPTION
**Pull request by evankp**
- [x] Create Login page
- Ignore Forget password (we don't need that right now)
- [x] Create Signup Page
- [x] There is no landing page - redirect to signup
- [x] Signup page - name, email, password + confirm password
- [x] Navbar - just for the logo for now, no buttons on the right hand side atm
- [x] Front-end validation as well

- Added Axios and @material-ui/icons packages to client
- Added prop-types package
- Added validator package

### CODE REVIEW

**Comment by evankp** [comment]
Still working on validation, so I will need to rework a couple of files related to that.

File: client/src/components/Inputs.js
Comment (DevDelvo) [inline client/src/components/Inputs.js:83]: a bit nitpicky but I think you can do `export const ComponentName` and you can still destructure it as normal. It can save some lines!

**Review by DevDelvo — COMMENTED** [review]
Looks good as far as I can tell!

File: client/src/components/Inputs.js
Comment (evankp) [reply client/src/components/Inputs.js:83]: Oh ya, slipped my mind for some reason, thanks

Comment (evankp) [reply client/src/components/Inputs.js:83]: Actually I just remembered why I am doing this, and I don't know if this actually affects anything other then IDEs but I am defining prop-types outside the component due to the components being functions instead of classes, so I need to put exports after CustomInputOutline.propTypes = {...}

**Review by DevDelvo — APPROVED** [review]
Looks good so far. Hope we can merge this soon after Taryn reviews it as well!

File: client/src/components/ProductLaunchPage.js
Comment (tsyaeger) [inline client/src/components/ProductLaunchPage.js]: I might be misunderstanding here, but why is this file ProductLaunchPage if it's the logo component?

File: client/src/pages/Login.js
Comment (tsyaeger) [inline client/src/pages/Login.js]: Just wanted to make sure that we are logging in with email vs username vs both. In backend I have just username at the moment.

File: client/src/components/ProductLaunchPage.js
Comment (evankp) [reply client/src/components/ProductLaunchPage.js]: Oh shoot I did mean for it to be logo I guess when I renamed the files I wasn't paying as much attention I'll fix that before I merge thanks

File: client/src/pages/Login.js
Comment (evankp) [reply client/src/pages/Login.js:45]: I think it is email, as by the ticket for sign up page says name, email and password, but I can change that when I do integration

**Pull request merged**
